.github/workflows/msvc-ci.yml

```diff
@@ -26,7 +26,7 @@ jobs:
     
     env:
       WINGTK_URL: https://github.com/wingtk/gvsbuild/releases/download/2024.10.0/GTK${{ matrix.gtk-major-version }}_Gvsbuild_2024.10.0_x64.zip
-      LUAINSTALLER_URL: https://github.com/luau-project/LuaInstaller/releases/download/v0.3.0.0/LuaInstaller.Console-v0.3.0.0-x64.zip
+      LUAINSTALLER_URL: https://github.com/luau-project/LuaInstaller/releases/download/v0.5.0.0/LuaInstaller.Console-v0.5.0.0-x64.zip
```

Comment:
What's this? How does this bring in libreadline?

Reply:
this is unrelated to ubuntu CI. Just updating links.